cobalt/watchdog/watchdog_test.cc

```diff
@@ -164,7 +164,7 @@ TEST_F(WatchdogTest, ViolationsJsonShouldPersistAndBeValid) {
                                   base::kApplicationStateStarted,
                                   kWatchdogMonitorFrequency));
   ASSERT_TRUE(watchdog_->Ping("test-name", "test-ping"));
-  SbThreadSleep(kWatchdogMonitorFrequency * 2);
+  SbThreadSleep(kWatchdogMonitorFrequency * 4);
```

Comment:
Perhaps move that 4 into a constant at the top of the file ?

Reply:
done